src/Registration/Repositories/Entities/RegistrationEntity.php

```diff
@@ -0,0 +1,135 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Registration\Repositories\Entities;
+
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Uuid;
+use VSV\GVQ_API\Common\Repositories\Entities\Entity;
+use VSV\GVQ_API\Registration\Models\Registration;
+use VSV\GVQ_API\User\Repositories\Entities\UserEntity;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="registration")
+ */
+class RegistrationEntity extends Entity
+{
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", name="hash_code", length=22, nullable=false)
+     */
+    private $hashCode;
+
+    /**
+     * @var UserEntity
+     *
+     * @ORM\OneToOne(targetEntity="VSV\GVQ_API\User\Repositories\Entities\UserEntity", fetch="EAGER")
+     * @ORM\JoinColumn(name="user_id", referencedColumnName="id", nullable=false)
+     */
+    private $userEntity;
```

Comment:
@stejes we kunnen hier misschien een unique constraint opzetten, dan kunnen er voor 1 user nooit 2 registraties zijn.